apps/tome-api/src/features/card/card.fns.ts

```diff
@@ -1631,4 +1630,513 @@ export const notImplementedCards: DbCard[] = [
+		effects: {},
+	},
+	{
+		id: '123',
+		name: 'Even the Odds',
+		type: 'spell',
+		colors: ['blue'],
+		attack: 15,
+		description: 'If this spell deals even damage, double the damage dealt. If it deals odd damage, discard this card.',
+		effects: {},
+	},
+	{
+		id: '124',
+		name: 'Odd the Evens',
+		type: 'spell',
+		colors: ['red'],
+		attack: 16,
+		description: 'If this spell deals odd damage, heal 7 HP. If it deals even damage, discard this card.',
+		effects: {},
+	},
+	// Archetype based on revealing a card from the opponent's hand and if it's a type or color something happens. I want the theme to be about scrying or looking into the future or looking into a crystal ball
+	{
+		id: '125',
+		name: 'Seer\'s Tent',
+		type: 'field',
+		color: 'blue',
+		description: 'At the start of each turn, each player reveals the top card of their deck then discard it. Depending on the color of the card, do the following: \
+		- Blue: Draw 1 card. \
+		- Red: Deal 5 damage to the opponent. \
+		- Green: Heal 5 HP. \
+		- Colourless: Take 10 damage.',
+		effects: {},
+	},
+	{
+		id: '126',
+		name: 'Crystal Ball',
+		type: 'spell',
+		colors: ['red', 'green', 'blue'],
+		attack: 0,
+		description: 'Draw 1 card and reveal it. Depending on the color of the card, do the following: \
+		- Blue: Deal 17 damage to the opponent. Then discard this card. \
+		- Red: Discard 1 card from your hand and deal 13 damage to the opponent. \
+		- Green: Deal 9 damage to the opponent and heal 9 HP.',
+		effects: {},
+	},
+	{
+		id: '127',
+		name: 'Scrying Orb',
+		type: 'spell',
+		colors: ['blue'],
+		attack: 5,
+		description: 'Draw 2 cards and discard 1 card from your hand.',
+		effects: {},
+	},
+	{
+		id: '128',
+		name: 'Tarot Reading: The Fool',
+		type: 'spell',
+		colors: ['green'],
+		attack: 8,
+		description: 'Discard one card from your opponent\'s hand. If the card is a BLUE card deal 12 damage to the opponent. If it is a GREEN card, return this card to your hand.',
+		effects: {},
+	},
+	{
+		id: '129',
+		name: 'Tarot Reading: The Magician',
+		type: 'spell',
+		colors: ['red', 'blue'],
+		attack: 1,
+		description: 'This card has an effect depending on the number of cards in this stack. \
+		- 2 card: This cards attack is 10. \
+		- 5 cards: This cards attack is 20. \
+		- 8 cards: This cards attack is 30.',
+		effects: {},
+	},
+	{
+		id: '130',
+		name: 'Tarot Reading: The Star',
+		type: 'spell',
+		colors: ['red', 'green'],
+		attack: 5,
+		description: 'If you win combat with this card, draw 3 cards then discard this card. If you lose combat with this card, draw 1 card then discard this card.',
+		effects: {},
+	},
+	{
+		id: '131',
+		name: 'Tarot Reading: The Moon',
+		type: 'spell',
+		colors: ['blue'],
+		attack: 8,
+		description: 'When you lose combat with this card, deal 5 damage to the oppenent.',
+		effects: {},
+	},
+	{
+		id: '132',
+		name: 'Tarot Reading: The Sun',
```

Comment:
this is my favourite card, it's so cool! 3 might be OP, but it looks very fun

Reply:
Yeah I want to try it though haha like thematically this was my inspiration:
The sun produces white light, which contains all colours. So it's a RED GREEN BLUE card. Then you draw three and discard three, the idea was to have a way that even if you have 0 cards (unlikely in our game as of this moment) you would still have a chance to get all effects.

Let's try it and if it's really broken we nerf it XD